cmd/api/routers/routers.go

```diff
@@ -109,7 +109,15 @@ func (c *Controller) GetResources(context *gin.Context) {
 		return
 	}
 
-	continueToken := pagination.CreateToken(lastId, lastDate)
+	// We can't know if there are more resources in the DB than the limit, because the number
+	// of resources that we know about is limited by the limit value in the first place.
+	// So we do know that if the number of resources is equal to the limit, it is possible that
+	// there are more resources stored. With some usage this would be true most of the time
+	continueToken := ""
+	if len(resources) == limit {
```

Comment:
I would say that it's better to do a `limit+1` and drop the last result instead of giving the user the idea that there are more resources when they aren't. This is likely to happen not too many times but I think it's better to be accurate in here, specially if anyone want to rely on the `continueToken` field for an automation.

The limit+1 approach is far more efficient than doing a separate `count()`.

What do you think?

Reply:
I understand that you are proposing to run the DB query with an increased limit value than the provided by the user, then to drop the resources the user didn't ask for so we can check if there are more resources than the user asked for.

If that is right, I think it is worth a try.

Reply:
yes, that's what I mean :)

Reply:
Marta, give it a look and see what do you think. I still need to do tests, so give me feedback on the implementation.